Makefile

```diff
@@ -17,6 +17,8 @@ test: clean .inline-deps test/resources/cider/nrepl/clojuredocs/export.edn
 quick-test: clean
 	lein with-profile -user,-dev,+$(CLOJURE_VERSION),+test test
 
+fast-test: quick-test
```

Comment:
I guess that's a bit of a misnomer, as it won't be faster if the tests pass. Perhaps `fail-fast-test`?

Reply:
You named the PR "Fail faster on CI" which got me confused. 

Then my comment would be the following - occasionally there are MrAnderson specific issues, which we won't catch if we don't run the CI using it.